keyboards/keebio/iris/keymaps/johnstegeman/rules.mk

```diff
@@ -0,0 +1,31 @@
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+SPLIT_KEYBOARD = yes
+ENCODER_ENABLE = no
+#RGB_MATRIX_ENABLE = yes
+#RGB_MATRIX_DRIVER = WS2812
+
+LTO_ENABLE = yes
+VIA_ENABLE = yes
+
+
+RGBLIGHT_ENABLE = yes # by default this is no, but for the rgb light layers to work it needs to be `yes`
+RGB_MATRIX_ENABLE = no # by default this is yes, but for the rgb light layers to work it needs to be `no` 
```

Comment:
You can do the same with rgb matrix, and the indicators function: 
https://docs.qmk.fm/#/feature_rgb_matrix?id=callbacks

It's a bit more manual, but just as flexible